waterloop-site/src/components/Sponsor/SponsorPage.tsx

```diff
@@ -26,6 +26,21 @@ import ArtsEndowmentFundLogo from '../../static/img/official-sponsors/ArtsEndowm
 import Crosslink from '../../static/img/official-sponsors/Crosslink.png'
 import SponsorModal from './SponsorModal';
 
+//Mock data for Supporters
+const MockSupportersData = [
+  "Supporter with a very long name",
+  "Supporter with a very long name",
+  "Supporter with a very long name",
+  "Supporter with a very long name",
+  "Supporter #1",
+  "Supporter #1",
+  "Supporter with a very long name",
+  "Supporter with a very long name",
+  "Supporter with a very long name",
+  "Supporter #1",
+  "Supporter with a very long name",
+  "Supporter #1",
+];
```

Comment:
Move these into the sponsors array

Reply:
add interface ISupporter:

```
interface ISupporter {
  name: string;
}
```

Then change sponsors to be: 
```
const sponsors: Array<ISponsor | ISupporter> = [
...
]
```

And add the list of supporters to the bottom of the sponsors array.